editor/editor_feature_profile.cpp

```diff
@@ -43,6 +43,7 @@
 const char *EditorFeatureProfile::feature_names[FEATURE_MAX] = {
 	TTRC("3D Editor"),
 	TTRC("Script Editor"),
+	TTRC("Game Editor"),
```

Comment:
Game Editor?
Maybe Game View? Or Game Screen? idk

Reply:
"Game View" is a far better name actually.